internal/handlers/pg/msg_count.go

```diff
@@ -75,9 +75,10 @@ func (h *Handler) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 
 	var ok bool
 	if sp.Collection, ok = collectionParam.(string); !ok {
-		return nil, common.NewErrorMsg(
+		return nil, common.NewCommandErrorMsgWithArgument(
```

Comment:
Command is not an argument.

The same is true everywhere, it seems

Reply:
Here I believe that command is an argument, because parameter which is storing the collection name is called as it's command name
```
	collectionParam, err := document.Get(document.Command())
```

Reply:
Command is never an argument :) We already track commands separately

Reply:
Then there's no argument in this example?